Mathlib/AlgebraicGeometry/PrimeSpectrum/TensorProduct.lean

```diff
@@ -0,0 +1,79 @@
+/-
+Copyright (c) 2024 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.RingTheory.SurjectiveOnStalks
+import Mathlib.AlgebraicGeometry.PrimeSpectrum.Basic
+
+/-!
+
+# Lemmas regarding the prime spectrum of tensor products
+
+## Main result
+- `PrimeSpectrum.embedding_tensorProductTo_of_surjectiveOnStalks`:
+  If `R →+* T` is surjective on stalks,
+  then `Spec(S ⊗[R] T) → Spec S × Spec T` is a topological embedding.
```

Comment:
Wait, don't we already have the statement that this is an iso? Or is it only an iso as raw types?